frontend/src/app/utils/api.ts

```diff
@@ -70,7 +70,12 @@ class FetchClient {
       this.getOptions(method, body)
     );
     if (!res.ok) {
-      throw await res.text();
+      const errorText = await res.text();
+      if (String(errorText).includes("Session timeout")) {
+        window.location.href = "/app/session-timeout";
```

Comment:
I wonder if adding a `return;` here would solve the error page appearing before the session timeout page? I think since there isn't a `return`, execution is continuing after the `if (!res.ok)` block, but I'm not sure if that's causing any issues.

Reply:
That does seem to have done the trick, thanks! It briefly flashes to the next page in the flow but that's better than the not found link in my opinion.
![SimpleReport (5)](https://user-images.githubusercontent.com/80282552/148618540-6806de6f-8f5e-4c4f-aab2-c91beef5aad2.gif)